server/routes/bigpanda/tasks.py

```
@@ -12,9 +12,116 @@
+    if alerts:
+        parts.append("")
+        parts.append("TOP CHILD ALERTS:")
+        for i, a in enumerate(alerts[:5], 1):
+            desc = a.get("description") or a.get("condition_name") or "no description"
+            src = a.get("source_system") or "unknown source"
+            parts.append(f"  {i}. [{src}] {desc}")
+
+    try:
+        from chat.background.rca_prompt_builder import inject_aurora_learn_context
+        inject_aurora_learn_context(parts, user_id, title, service, "bigpanda")
+    except Exception as exc:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)